Sources/Overload/OvCore/premake5.lua

```diff
@@ -11,12 +11,10 @@ project "OvCore"
 	objdir (objoutdir .. "%{cfg.buildcfg}/%{prj.name}")
 	characterset ("MBCS")
 	
-	buildoptions { "/bigobj" }
```

Comment:
This flag is required with MSVC when compiling certain file requiring a lot of memory

Reply:
Possible solution woulf to be to check which compiler is used and then add this flag only if required? I do not have the premake knowledge tho